extensions/ql-vscode/src/language-support/contextual/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from "../ast-viewer/ast-builder";
```

Comment:
Hmm this doesn't look right, why is it needed?

Reply:
Come to think of it, while is there another index.ts file here? I was under the impression we were going to have 1 at each top-level directory (unless there were other reasons that make it difficult, like in a previous PR).